web/src/modules/i18n/BrowserLanguageDetector.js

```diff
@@ -0,0 +1,65 @@
+let hasLocalStorageSupport = null
+
+const localStorageAvailable = () => {
```

Comment:
I copied this from the external module. I'd keep it as it is to avoid problems with browsers. This code is quite well tested.

Reply:
Why did you chose to copy this? Probably it would be good to add a comment. Did you modify something? Should we include the tests as well then?

Reply:
I did some adjustments so I think there is no reason to add a comment. I create tests myself.

Reply:
Hmm your call. I think for future reference it wouldn't be bad but never mind :shrug: